lib/helper/colon.dart

```diff
@@ -0,0 +1,5 @@
+import 'package:flutter/cupertino.dart';
+
+Text addColon(String object, String value) {
```

Comment:
This should be called something like "displayText" as addColon is just an implementation detail.